website/widgets/form/inline.md

```diff
@@ -22,3 +22,18 @@ q.page['form'] = ui.form_card(
 ```
 
 Check the full API at [ui.inline](/docs/api/ui#inline).
+
+## Customize the alignment
+
+### Horizontal alignment (`justify`)
+
+You can specify how the elements should be horizontally aligned using the `justify` parameter. The default value is `start`.
+
+![Supported values for the justify parameter](/img/widgets/inline-justify.svg)
```

Comment:
Instead of adding the images yourself, you can write a minimal code (see L17 of this file) and during the build, the code gets executed and a corresponding screenshot for it is generated and appended. This way people can directly see what gets rendered after copypasting the code.

It's enough if you just include a few code examples (one per align/justify).

If you are curious though, you can try the build yourself:

* From project root run `make build-ui` to create a UI build including your most recent changes.
* `make pydocs` which will generate all the docs assets.
* `cd website && npm start` to see the changed docs.

Make sure to remove your custom `svg` files from the PR once you are done.